news post for new global team members (#289)
rladies/rladies.github.io: content/news/2024-02-07-welcome-global-team/index.en.md

| ![](img/sara_acevedo.jpg)           | Sara Acevedo              | Code of Conduct    |
| ![](img/hebah_bukhari.jpg)          | Hebah Bukhari             | Community Slack    |
| ![](img/priyanka_gagneja.jpg)       | Priyanka Gagneja          | Community Slack    |
| ![](img/renata_hirota.png)          | Renata Hirota             | Blog               |
| ![](img/virginia_garcia_alonso.jpg) | Virginia A. García Alonso | Mentoring          |
| ![](img/nic_crane.jpg)              | Nic Crane                 | Meetup Pro         |
| ![](img/alyssa_columbus.png)        | Alyssa Columbus           | Chapter Onboarding |
| ![](img/luana_atunes.png)           | Luana Atunes              | Abstract Review    |
| ![](img/sayantika_banik.png)        | Sayantika Banik           | Abstract Review    |
| ![](img/cosima_meyer.png)           | Cosima Meyer              | Website            |
| ![](img/andrea_gomez_vargas.png)    | Andrea Gómez Vargas       | Website            |
| ![](img/leena_el_seed.jpg)          | Leena El Seed             | Conference Liaison |
| ![](img/daniela_vazquez.png)        | Daniela Vázquez           | Social Media       |

# Onboarding process

> 

Reviewer: ```suggestion
The global governance of R-Ladies is seeing increased activity as we all  slowly are
coming out of 4 difficult years. The new members of the Global team are vital to
help support and grow our community.

```

| ![](img/hebah_bukhari.jpg)          | Hebah Bukhari             | Community Slack    |
| ![](img/priyanka_gagneja.jpg)       | Priyanka Gagneja          | Community Slack    |
| ![](img/renata_hirota.png)          | Renata Hirota             | Blog               |
| ![](img/virginia_garcia_alonso.jpg) | Virginia A. García Alonso | Mentoring          |
| ![](img/nic_crane.jpg)              | Nic Crane                 | Meetup Pro         |
| ![](img/alyssa_columbus.png)        | Alyssa Columbus           | Chapter Onboarding |
| ![](img/luana_atunes.png)           | Luana Atunes              | Abstract Review    |
| ![](img/sayantika_banik.png)        | Sayantika Banik           | Abstract Review    |
| ![](img/cosima_meyer.png)           | Cosima Meyer              | Website            |
| ![](img/andrea_gomez_vargas.png)    | Andrea Gómez Vargas       | Website            |
| ![](img/leena_el_seed.jpg)          | Leena El Seed             | Conference Liaison |
| ![](img/daniela_vazquez.png)        | Daniela Vázquez           | Social Media       |

# Onboarding process
The global governance of R-Ladies is seeing increased activity as we all  slowly are

> coming out of 4 difficult years. The new members of the Global team are vital to

Author: ```suggestion
coming out of four difficult years. The new members of the Global team are vital to
```

help support and grow our community.

With these amazing new members, we have also been able to develop
a new onboarding system for the Global Team. By using GitHub actions
and issues, we hope that the onboarding process will be more streamlined
in the future.
While the current new members did experience this system in its infancy,
we hope that it will be a smoother process for future members.

# What the future holds

We are excited to see what the future holds for R-Ladies with these new
members of the Global Team. Having new people, with fresh ideas and 
perspectives, will help us to continue to grow and improve as an organization.
We are looking forward to working with these new members, and we hope
that you will join us in welcoming them to the Global Team!

> You can see an overview of the entire global team [here](/about-us/global-team/).

Author: ```suggestion
You can see an overview of the entire global team at [About -> Meet the Global Team](/about-us/global-team/).
```

---
title: "Welcome to the Global Team"
author: "Leadership Team"
type: blog
date: "2024-02-07"
output: html_document
image: img/welcome.PNG
tags: 
- community
categories: 
- R-Ladies

> summary: R-Ladies Global Team has 13 new members!

Reviewer 2: ```suggestion
summary: R-Ladies Global Team has 15 new members!
```

slug: /news/new-to-global-team-2023/
---

> ![Thumbnail photos of 13 new global team members on purple background with text `Welcome`.](img/welcome.PNG)

Reviewer 2: ```suggestion
![Thumbnail photos of 15 new global team members on purple background with text `Welcome`.](img/welcome.PNG)
```

<!---
link to ppt that contains this image on rladies google account
https://docs.google.com/presentation/d/1ZnCmaO_gBSKSbtx_1tTicQSxnTJZaYGc/edit#slide=id.p1
-->

# Overview

In 2023 we [announced openings](https://rladies.org/news/2023-04-11-global-team-recruiting/) for roles on the R-Ladies Global Team that support organizational efforts and facilitate the growth of the R-Ladies community.
We are delighted to welcome 15 new members to the R-Ladies Global Team.
R-Ladies is 100% volunteer-driven, and we are very grateful to those who 
support us with their time and effort to champion our [mission](https://rladies.org/about-us/mission/).

# New members

|                                     |                           |                    |
|-------------------------------------|---------------------------|--------------------|
| ![](img/glenda_mendieta.jpg)        | Glenda Mendieta           | Translation        |

> | ![](img/sara_acevedo.jpg)           | Nicola Rennie             | Campaigns          |

Reviewer: ```suggestion
| ![](img/nicola_rennie.jpg)           | Nicola Rennie             | Campaigns          |
```

| ![](img/sara_acevedo.jpg)           | Sara Acevedo              | Code of Conduct    |
| ![](img/hebah_bukhari.jpg)          | Hebah Bukhari             | Community Slack    |
| ![](img/priyanka_gagneja.jpg)       | Priyanka Gagneja          | Community Slack    |
| ![](img/renata_hirota.png)          | Renata Hirota             | Blog               |
| ![](img/virginia_garcia_alonso.jpg) | Virginia A. García Alonso | Mentoring          |

> | ![](img/nicola_rennie.jpg)          | Nic Crane                 | Meetup Pro         |

Reviewer: ```suggestion
| ![](img/nic_crane.jpg)          | Nic Crane                 | Meetup Pro         |
```

| ![](img/alyssa_columbus.png)        | Alyssa Columbus           | Chapter Onboarding |
| ![](img/luana_atunes.png)           | Luana Atunes              | Abstract Review    |
| ![](img/sayantika_banik.png)        | Sayantika Banik           | Abstract Review    |
| ![](img/cosima_meyer.png)           | Cosima Meyer              | Website            |
| ![](img/andrea_gomez_vargas.png)    | Andrea Gómez Vargas       | Website            |
| ![](img/leena_el_seed.jpg)          | Leena El Seed             | Conference Liaison |
| ![](img/daniela_vazquez.png)        | Daniela Vázquez           | Social Media       |

# Onboarding needs

> Between 2020-2023, membership R-Ladies global organization increased 
> while the availability of volunteers remained limited. At the conclusion of this

Author: ```suggestion
Between 2020-2023, membership in the R-Ladies global organization increased 
while the availability of volunteers remained limited. At the conclusion of this
```
